File: 12_arrays_to_map/arrays_to_map.py
arr1 = ["abc", 3, "yo"]
arr2 = [42, "wassup", True]

def arrays_to_map(arr1, arr2):
	map = {}
	if len(arr1) != len(arr2):
		return None

	for i in range(len(arr1)):
		map[arr1[i]] = arr2[i]

	return map

result = arrays_to_map(arr1, arr2)
print(result)

File: 12_arrays_to_map/arrays_to_map.js
var arr1 = ["abc", 3, "yo"];
var arr2 = [42, "wassup", true];

function arrays_to_map(arr1, arr2) {
	var map = {};
	if (arr1.length != arr2.length) {
		return null;
	}
	for (var i = 0; i < arr1.length; i++) {
		map[arr1[i]] = arr2[i];
	}
	return map;
}

var result = arrays_to_map(arr1, arr2);
console.log(result);
